contracts/L1/gateway/L1Migrator.sol

```diff
@@ -108,6 +108,8 @@ contract L1Migrator is
         MigrateSenderParams params
     );
 
+    event L2MigratorUpdate(address _l2Migrator);
```

Comment:
```suggestion
    event L2MigratorUpdate(address l2Migrator);
```

since the convention for event params has not used underscores previously.

Reply:
amended the original commit to address this change.